libraries/AP_HAL_ChibiOS/hwdef/FlywooF745/README.md

```diff
@@ -0,0 +1,102 @@
+# Flywoo GOKU GN 745 40A AIO
+
+https://flywoo.net/products/goku-gn745-40a-aio-bl_32-mpu6000-25-5-x-25-5
+
+The Flywoo GOKU GN 745 AIO is a flight controller produced by [Flywoo](https://flywoo.net/).
+
+## Features
+
+- MCU: STM32F745 32-bit processor,216MHz, 512Kbytes Flash
+- IMU: MPU6000 (SPI)
+- Barometer: BMP280
+- Onboard LED：WS2812*4
+- USB VCP Driver (all UARTs usable simultaneously; USB does not take up a UART)
+- 7 hardware UARTS (UART1,2,3,4,5,6,7)
+- Supports serial receivers (SBUS, iBus, Spektrum, Crossfire) only.
+- PPM and PWM receivers are not supported.
+- Onbord 8Mbytes for Blackbox logging
+- 9V Power Out: 1.5A max
+- 5V Power Out: 2.0A max
+- 3.3V Power Out: 0.5A max
+- Dimensions: 33.5x33.5mm
+- Mounting Holes: Standard 25.5/26.5mm square to center of holes 
+- Weight: 8.5g
+
+- Built-in 40A BL_32 4in1 ESC
+- Support BLheli / BLHELI_32
+- Support PWM, Oneshot125, Oneshot42, Multishot, Dshot150, Dshot300, Dshot600,Dshot1200
+- Input Voltage: 2-6S Lipo
+- Continuous Current: 40A
+- Firmware: BLHELI_32
+
+## Pinout
+
+![GOKU GN 745 40A AIO](GOKUGN745AIO_Pinout.png "GOKU GN 745 40A AIO")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+|Name|Pin|Function|
+|:-|:-|:-|
+|SERIAL0|COMPUTER|USB|
+|SERIAL1|RX1/TX1|UART1|
+|SERIAL2|TX2/RX2|UART2|
+|SERIAL3|TX3/RX3|UART3 (RC Input/Output)|
+|SERIAL4|TX4/RX4|UART4|
```

Comment:
default is GPS

Reply:
I would like to make the documentation all match. Since the Flywoo diagram shows GPS on UART 6, should I set "SERIAL4_Protocol -1" in the parm file, and leave line 46 of the README.md the way it currently is?